app/views/shared/_comments.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if (post.comments.empty?) %>
+  <p>No comment yet.</p>
+<% else %>
+  <% post.comments.each do |comment| %>
```

Comment:
- [ ] Here, you should use your recent_comments method to render the 5 most recent comments 👍

Reply:
Thanks.